pkg/restapi/webhook/webhook.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webhook
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+)
+
+type webhookRequest struct {
+	Name        string
+	Destination string
+}
+
+var logger = log.New("aries-framework/agentd")
+
+// UnexpectedExitErrMsg is the string to be displayed when the webhook server exits unexpectedly.
+const UnexpectedExitErrMsg = "webhook server exited unexpectedly"
```

Comment:
error constant?

var UnexpectedExitErr = errors.New("webhook server exited unexpectedly")

Reply:
note error variables are prefixed with Err

Reply:
ErrUnexpectedExit

Reply:
I made it a var and changed it to ErrUnexpectedExitMsg, but I can't change it to an error type since I need it to be a string.

Reply:
With my new changes, this var isn't used anymore, and so I have deleted it.